version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version is the version of the build.
```

Comment:
Just to be clear, might be nice to add "CRI-O" before build in the comment here.